routes/groups.js

```diff
@@ -6,121 +6,87 @@ const cloudinary = require('cloudinary').v2;
 const fs = require('fs');
 
 const Group = require('../models/groups');
-const Sport = require('../models/sports');
-const User = require('../models/users');
+const GroupService = require('../services/Group');
 
-router.post('/create', (req, res) => {
+router.post('/create', async(req, res) => {
 
-    let { token, photo, name, sport, maxMembers, genders, levels, ageMin, ageMax, description, label, latitude, longitude } = req.body;
+    let { token, photo, name, sport, maxMembers, genders, levels, ageMin, ageMax, description, label, latitude, longitude, status } = req.body;
 
-
-    if (!token && !photo && !name && !sport && !maxMembers && !genders && !levels && !ageMin && !ageMax && !description && !label && !latitude && !longitude) {
-        res.json({ result: false, message: 'Missing or empty fields.' });
+    if (!token || !photo || !name || !sport || !maxMembers || !genders || !levels || !ageMin || !ageMax || !description || !label || !latitude || !longitude) {
```

Comment:
`status` is not mandatory ?

Reply:
ok but it's not in the list :p

Reply:
it's set by default in the users model (see backend > models > users.js)